backend/core/validations/user_validation/NameValidation.js

```diff
@@ -0,0 +1,9 @@
+function validateName(firstName, secondName) {
```

Comment:
I would either rename this function into validateNames, or just call it twice and only have it validate one parameter inside of it. The latter is prefered by me.